lib/audit/checksum.rb

```diff
@@ -12,6 +12,7 @@ def self.validate_disk(endpoint_name, limit=Settings.c2m_sql_limit)
     # batches.  we can't use ActiveRecord's .find_each, because that'll disregard the order .fixity_check_expired
     # specified.  so we use our own batch processing method, which does respect Relation order.
     pcs_w_expired_fixity_check = PreservedCopy.by_endpoint_name(endpoint_name).fixity_check_expired
+    logger.info "Preserved Copies to be checksum validated: #{pcs_w_expired_fixity_check.count}"
```

Comment:
How about "Number of Preserved Copies"? or "Count of"? It would make it clearer to me that I'm not looking at, for example, a single database id...